db/migrate/20160228140111_update_params_priority.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateParamsPriority < ActiveRecord::Migration
+  def change
+    OsParameter.update_all(:priority => 3)
```

Comment:
The base class has a [`default_scope`](https://github.com/theforeman/foreman/blob/develop/app/models/parameter.rb#L16), which causes the SQL generated by this to be:
```sql
UPDATE "parameters" SET "priority" = 3 WHERE "parameters"."id" IN (SELECT "parameters"."id" FROM "parameters"  WHERE "parameters"."type" IN ('OsParameter')  ORDER BY parameters.name)
```
while using `unscoped` leads to the following SQL:
```sql
UPDATE "parameters" SET "priority" = 3 WHERE "parameters"."type" IN ('OsParameter')
```